examples/notebooks/FastaiLRFinder_MNIST.ipynb

```diff
@@ -251,11 +308,10 @@
    },
    "outputs": [],
    "source": [
-    "optimizer.param_groups[0]['lr'] = lr_finder.lr_suggestion()\n",
+    "new_trainer.run(trainloader, max_epochs=10)\n",
```

Comment:
I think it is better to reduce `max_epochs` to 5 epochs everywhere. In case of suggested LR it is rather large and on during the last 5 epochs batch loss jumps too much as LR is rather high.